mozilla/blurts-server
# MNTOR-1327: redirect dashboard

# References: 
Jira: MNTOR-1327
Figma: 




# Description
for v2 release, we want to temp redirect users from dashboard to breach details page for backwards compatibility reasons. Old emails still have a reference to the dashboard URI.


# Screenshot (if applicable)

Not applicable.

# How to test
If you have an old email, click on the dashboard link.
If you don't have an old email, go to `/user/dashboard/` and see the redirect for `/user/breaches/`

# Checklist (Definition of Done)
